controllers/providers/aws/aws.go

```diff
@@ -700,3 +720,164 @@ func IsNodeGroupInConditionState(key string, condition string) bool {
 		return false
 	}
 }
+
+func IsStackInConditionState(key string, condition string) bool {
```

Comment:
Since we removed the cloudformation provisioner, I think we can remove this one.
Are you using this anywhere?

Reply:
no, it seems to an orphaned function.

Reply:
removed it